client/src/main/java/io/pravega/client/state/impl/StateSynchronizerImpl.java

```diff
@@ -243,7 +245,7 @@ private boolean isNewer(Revision revision) {
     @Synchronized
     private void updateCurrentState(StateT newValue) {
         if (newValue != null && isNewer(newValue.getRevision())) {
-            log.trace("Updating new state to {} ", newValue.getRevision());
+            log.debug("Updating new state to {} ", newValue.getRevision());
```

Comment:
This log isn't useful enough on its own to be debug. It's not logging the segment name that is updated so you would need trace logs to see what the thread was updating. 

Let's try to keep it to one log line per successful update at debug that contains all the info we would want.

Reply:
yes, this is redundant, will change it to trace.